### PR TITLE
[libopenmpt] Update to 0.6.7

### LINKED ISSUE #28930: [libopenmpt] update to 0.6.7
Library name: libopenmpt

New version number: 0.6.7

Other information that may be useful (release notes, etc...)
The existing version is one major release number below the current one. Even tho it was updated with some security patches, it's still old.

### PR DESCRIPTION
**Pull request by unjello**
Closes #28930

**Describe the pull request**

- #### What does your PR fix?
  Updates libopenmpt to 0.6.7

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all (although I've only been able to test it on arm64-mac, the changes SHOULD be transparent as it's only a version bump and upstream lib compiles in their CI). It's my first PR to vcpkg so PLEASE, be gentle :)

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
 Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by unjello** [comment]
> microsoft-github-policy-service agree

microsoft-github-policy-service agree

**Comment by unjello** [comment]
> @unjello please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
>  > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by Sibras** [comment]
This is not necessarily an issue introduced by this PR (as its been there a while), but the created openmpt pkg-config file is broken as it contains the incorrect lib names for openmpt and dependency libs (especially in debug builds)

**Review by vicroms — DISMISSED** [review]

File: ports/libopenmpt/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "libopenmpt",
-  "version": "0.5.12",
-  "description": "a library to render tracker music",
+  "version": "0.6.7",
+  "description": "a cross-platform C++ and C library to decode tracked music files (modules) into a raw PCM audio stream.",
   "homepage": "https://openmpt.org/",
```
Comment (MonicaLiu0311) [inline ports/libopenmpt/vcpkg.json:5]:
```suggestion
  "version": "0.6.7",
  "description": "a cross-platform C++ and C library to decode tracked music files (modules) into a raw PCM audio stream.",
  "homepage": "https://openmpt.org/",
  "license": "BSD-3-Clause",
```

A `license` is required.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**